### ISSUE TITLE
Maybe a bug in an aquaplanet when setting land temperature values in ForcingData

### ISSUE THREAD

**Issue by meteorologytoday** [climate-analytics-lab/jax-gcm#318]
When I try to run an aquaplanet simulation with the code 
```
import jax.numpy as jnp
import jcm
from jcm.model import Model
from jcm.forcing import ForcingData

def repeat(arr, repeats=365):
    return jnp.repeat(jnp.expand_dims(arr, axis=-1), axis=-1, repeats=repeats)

model = Model(
    time_step=30.0,
)

D2_nodal_shape = model.coords.nodal_shape[1:]
forcing = ForcingData(
    alb0 = jnp.zeros(D2_nodal_shape),
    sice_am = repeat(jnp.zeros(D2_nodal_shape)),
    snowc_am = repeat(jnp.zeros(D2_nodal_shape)),
    soilw_am = repeat(jnp.zeros(D2_nodal_shape)),
    stl_am = repeat(jnp.zeros(D2_nodal_shape)),
    sea_surface_temperature = repeat(jnp.zeros(D2_nodal_shape) + 290.15),
    lfluxland = True,
)

# Run simulation
predictions = model.run(
    forcing=forcing,
    save_interval=1.0,   # save every day
    total_time=10.0      # 10-day simulation
)

# Convert to xarray and save
ds = predictions.to_xarray()
ds.to_netcdf("output.nc")
```
it results in output `surface_flux.tskin` being nan.

If set 
```
lfluxland = False,
```
then `surface_flux.tskin` becomes all zeros.

If set
```
stl_am = repeat(jnp.zeros(D2_nodal_shape) + 288.15),
lfluxland = False,
```
then things will work okay.

I am guessing this is not a desired behavior?

**Comment by duncanwp** [comment]
OBE: the repro's API is gone — lfluxland moved to TerrainData and is auto-set from the land mask (all-ocean disables the land branch), and the aquaplanet/zeros factories default land temperature to a physical value rather than 0 K.
